src/receivers/custom-routes.ts

```diff
@@ -13,7 +13,7 @@ export interface ReceiverRoutes {
   };
 }
 
-export function prepareRoutes(customRoutes: CustomRoute[]): ReceiverRoutes {
+export function buildReceiverRoutes(customRoutes: CustomRoute[]): ReceiverRoutes {
```

Comment:
Either is fine but, to me, the name `build` indicates the nuance that this method does not side effects much more.

Reply:
I have always had a problem with using `prepare` to describe functionality. Unless one is familiar with the algorithm already, it is not descriptive to an outsider!

I am OK with `buildReceiverRoutes`. I also think `attachReceiverRoutes` or `bindReceiverRoutes` would be acceptable.

Reply:
Thanks for the suggestions. I think that we can go with "build" for this